packages/react/src/hooks/useUserAttributes.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ *  This version of useUserAttributes is functional, but there are several type errors relating to the handleActions. Also, the handleActions are hardcoded.
+ */
+
+import { Hub, HubCallback } from "@aws-amplify/core";
+
+import { useDataState } from "./useDataState";
+import {
+  DefaultAttributes,
+  defaultSendUserAttributeVerificationCodeOutput,
+  defaultUpdateUserAttributesOutput,
+} from "./constants";
+
+import {
+  HandlerInputs,
+  StateDataTypes,
+  confirmUserAttributeAction,
+  deleteUserAttributesAction,
+  fetchUserAttributesAction,
+  sendUserAttributeVerificationCodeAction,
+  updateUserAttributesAction,
+  Actions,
+} from "./actionTypes";
+import React from "react";
```

Comment:
External deps should be imported before imports from our packages, followed by local imports. See https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#imports (Could probably use some updating, feel free to reach out offline with questions)